recipes/sphinxcontrib-restbuilder/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "sphinxcontrib-restbuilder" %}
+{% set version = "0.1" %}
+{% set sha256 = "8f2d7d73930fdedc3571adab32fbe843b4716829a291dbb27bab56b7c8d1e23d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sphinx
+    - six
+    - jinja2
+    - pygments
+    - docutils
+    - snowballstemmer
+    - babel
+    - alabaster
+    - imagesize
+    - requests
+    - sphinxcontrib-websupport
+    - colorama
+    - markupsafe
+    - pytz
+    - chardet
+    - idna
+    - urllib3
+    - certifi
+  run:
+    - python
+    - six
+    - jinja2
+    - pygments
+    - docutils
+    - snowballstemmer
+    - babel
+    - alabaster
+    - imagesize
+    - requests
+    - sphinxcontrib-websupport
+    - colorama
+    - markupsafe
+    - pytz
+    - chardet
+    - idna
+    - urllib3
+    - certifi
+
+test:
+  imports:
+    - sphinxcontrib.writers
+    - sphinxcontrib.builders
+
+about:
+  home: https://bitbucket.org/birkenfeld/sphinx-contrib
+  license: BSD 2-clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Sphinx extension to output reST files.'
+
+  description: |
+    Sphinx extension to build reST (reStructuredText) files.
+    This extension is in particular useful to use in combination with the autodoc extension to automatically generate documentation for use by any rst parser (such as the GitHub wiki).
```

Comment:
Please break this up.

Reply:
Done and build requirements reduced to sphinx